HWLesson 2.2/ALevelSample/Entities/BillEntity.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Models;
+
+namespace Entities
+{
+    public class BillEntity
+    {
+        public string Id { get; set; }
+        public string Number { get; set; }
+        public int Cost { get; set; }
+        public string UserID { get; set; }
+        public DateTime Date { get; set; }
```

Comment:
When you use DateTime it's a good practise to add 'at' in the end, like: CreatedAt